test/Chainweb/Test/RestAPI/Utils.hs

```diff
@@ -267,6 +283,10 @@ polling sid cenv rks pollingExpectation =
       Just cr ->  _crReqKey cr == rk && validate (_crResult cr)
       Nothing -> False
 
+getCurrentBlockHeight :: ChainwebVersion -> ClientEnv -> ChainId -> IO BlockHeight
+getCurrentBlockHeight сv cenv cid = do
+  cuts <- either (const $ error "failed to get cuts") id <$> runClientM (cutGetClient сv) cenv
+  return $ fromJust $ _bhwhHeight <$> HM.lookup cid (_cutHashes cuts)
```

Comment:
the error here is really harsh

Reply:
Introduced `GetBlockHeightFailure` for this, thanks!